pkg/key/key.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// Package key implements key manager and helper functions.
+package key
+
+import (
+	"bytes"
+	"errors"
+	"sort"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/vms/components/avax"
+)
+
+var (
+	ErrInvalidType = errors.New("invalid type")
+	ErrCantSpend   = errors.New("can't spend")
+)
+
+// Key defines methods for key manager interface.
+type Key interface {
+	// P returns all formatted P-Chain addresses.
```

Comment:
Huh? The interface is completely empty?

Reply:
whoops, I accidentally deleted, restored